elasticsearch/build.gradle.kts

```diff
@@ -33,6 +33,7 @@ tasks {
 
 micronautBuild {
     binaryCompatibility {
-        enabled.set(false)
+        enabled.set(true)
+        baselineVersion.set("5.0.0-M6")
```

Comment:
is the baselineVersion necessary? Can we just delete this whole block?

Reply:
no, thought it was at first but another build worked without